examples/distributed-solver/distributed-solver.cpp

```diff
@@ -72,22 +73,19 @@ int main(int argc, char* argv[])
     // non-distributed program. Please note that not all solvers support
     // distributed systems at the moment.
     using solver = gko::solver::Cg<ValueType>;
+    using schwarz =
+        gko::experimental::distributed::preconditioner::Schwarz<ValueType,
+                                                                LocalIndexType>;
+    using bj = gko::preconditioner::Jacobi<ValueType, LocalIndexType>;
```

Comment:
Not uses. I guess that is some leftover from when this example had a preconditioner?